bin/cc.sh

```diff
@@ -7,6 +7,6 @@ goroot="$(dirname "$bindir")"
 what=${1:-test}
 shift
 
-which -s spy || go get github.com/jpillora/spy
```

Comment:
The -s suppresses the output, which I require. If -s isn't on linux then how about `which spy >/dev/null`.